src/utils/helper-functions/versions.ts

```diff
@@ -0,0 +1,55 @@
+import semver from 'semver'
+
+const noopVersion = '0.0.0'
+const versionsCache: Map<string, string> = new Map()
+const satisfiesCache: Map<string, boolean> = new Map()
+
+function getNgrxVersion(pkg: string): string {
+  if (!versionsCache.has(pkg)) {
+    const version = readPlatformVersion(pkg)
+    versionsCache.set(pkg, version ?? noopVersion)
+  }
+
+  return versionsCache.get(pkg) as string
+}
+
+function readPlatformVersion(pkg: string) {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const ngrxVersion = require(`${pkg}/schematics-core`) as {
+      platformVersion?: string
+    }
+    const version = ngrxVersion.platformVersion
+      ?.split('')
+      .filter((c) => (c >= '0' && c <= '9') || c === '.')
+      .join('')
+    return version
```

Comment:
If I'm understanding it correctly, this strips all non-numeric + dots... what about:

```suggestion
    return ngrxVersion.platformVersion?.replace(/[^\d\.]/g, '')
```